.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build git-stat
+
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
@IlyaMoskva we have `master` primary branch

Reply:
Good catch, @g4s8 , Thanks. Fixed.